src/actions/clusterActions.js

```diff
@@ -632,24 +658,27 @@ export function clusterCreateKeyPair(clusterId, keypair) {
       keypair,
     });
 
-    var keypairsApi = new GiantSwarm.KeyPairsApi();
+    const keypairsApi = new GiantSwarm.KeyPairsApi();
+
     return keypairsApi
       .addKeyPair(clusterId, keypair)
-      .then(keypair => {
+      .then(pair => {
```

Comment:
Just curious, why rename this?

Reply:
Because it has the same name as one of the function arguments, and although it's not a problem due to scoping, it can cause confusion, and that's the reason for having a linter rule for this